src/models/Team.ts

```diff
@@ -1,6 +1,7 @@
 interface Team {
     readonly name: string,
     readonly id?: number
+    readonly image?: string
```

Comment:
Please don't change the main `Team` interface, it's used in almost all endpoints. I've decided against returning team logos since you can get any team image by replacing the team id in the following format: `https://static.hltv.org/images/team/logo/${teamID}` (the library exports the `https://static.hltv.org` as a property so it's even easier for the people)